.github/workflows/kuksa_feeders_docker.yml

```diff
@@ -0,0 +1,45 @@
+name: kuksa_feeders_docker
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Fine for now, in the future we should discuss whether we als should run this on our own runners